WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -22,21 +23,34 @@ class GutenbergSettings {
 
     // MARK: Public accessors
 
-    func isGutenbergEnabled() -> Bool {
-        return database.object(forKey: gutenbergEditorEnabledKey) as? Bool ?? false
+    /// True if gutenberg editor is currently enabled
+    var isGutenbergEnabled: Bool {
+        get {
+            return database.bool(forKey: Key.enabled)
+        }
+        set(isEnabled) {
+            if isGutenbergEnabled != isEnabled {
+                trackSettingChange(to: isEnabled)
```

Comment:
`WPAnalytics.refreshMetadata()` will read from `isGutenbergEnabled` to set the `gutenberg_enabled` property, so it needs to be called after the new value is set

Reply:
That's right! My bad 🙏 
Fixed in the last commit.